### DISCUSSION TITLE
40mm grenades can not be inserted in detonator

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15130] [Ideas and suggestions]
Detonator is multipurpose device. It can detonate uex, grenades, foam grenades.
40mm grenades can be used how more weak alternative to set monster traps inside a sub. Why then 40mm grenades can not be inserted in the detonator?

**Comment by jigardi** [comment]
I was wondering about this aswell, I am assuming it is due to some inconsistency in how their xmls are defined vs the other objects.

Perhaps including the xmls for the different objects here could help the community/devs figure out why this disparity exists;  and hopefully how to fix it.
